examples/case_studies/reliability_and_calibrated_prediction.myst.md

```diff
@@ -358,9 +361,9 @@ def plot_cdfs(actuarial_table, dist_fits=True, ax=None, title="", xy=(3000, 0.5)
     )
     ax.plot(actuarial_table["t"], actuarial_table["CI_95_ub"], color="darkorchid", linestyle="--")
     ax.fill_between(
-        actuarial_table["t"],
-        actuarial_table["CI_95_lb"],
-        actuarial_table["CI_95_ub"],
+        actuarial_table["t"].values,
```

Comment:
I had an error in matplotlib trying to run this, tried updating matplotlib and numpy and it didn't seem to fix the issue, so I added this values to continue ahead. I don't think it is a big issue though.

Reply:
Yeah, i think that's fine.